Jenkinsfile_CNP

```diff
@@ -75,6 +81,12 @@ withPipeline(type , product, component) {
         env.S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
         env.MP4_52MB = "e58a9fa5-4f6f-438f-b44a-8c0877e945d0/binary"
         env.MP4_111MB = "50f9dd86-1615-461d-bca8-e437103dd734/binary"
+        env.MP4_260KB = "267KB_mp4_video.mp4"
+        env.MP4_272MB = "272MB_mp4_video.mp4"
+        env.MP4_625MB = "652MB_video_mp4.mp4"
+        env.PDF_1KB = "1KB.pdf"
+        env.PDF_500MB = "500MB.pdf"
+        env.PDF_1_2GB = "1.2GB.pdf"
```

Comment:
why do we need these here? can not we can put into functional test?

Reply:
Moved to LargeMultiMediaUploadIT